cellect_env.rb

```diff
@@ -12,9 +12,9 @@ def setup_env_vars
     setup_environment
     preload_workflows
     load_db_yaml
-    load_zk_yaml
+    load_redis_yaml
     @env_vars = {
-      "ZK_URL" => @zk_url,
+      "ATTENTION_REDIS_URL" => @redis_url,
```

Comment:
can we use the redis_url method here instead to allow an explicit ENV var to override any yaml / other env configs?